superbench/analyzer/rule_base.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Do we need to add test for `rule_base.py`?

Reply:
i think the functions in rule_base have been tested in the test for DataDiagnosis